modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/resource-actions/default.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<!DOCTYPE resource-action-mapping PUBLIC "-//Liferay//DTD Resource Action Mapping 7.4.0//EN" "http://www.liferay.com/dtd/liferay-resource-action-mapping_7_4_0.dtd">
+
+<resource-action-mapping>
+	<portlet-resource>
+		<portlet-name>com_liferay_frontend_data_set_views_web_internal_portlet_FDSViewsPortlet</portlet-name>
+		<permissions>
+			<supports>
```

Comment:
We did not have this before, however, it looked to work, I've made a quick check, default ones are (snapshot taken from master):
![image](https://github.com/liferay-frontend/liferay-portal/assets/861014/d3e2b7df-8daf-40ef-8e20-3f1916543ec4)

Comparing with other portlet-resources for admin widgets, this one looks pretty similar and the system seems smart enough to generate the `PERMISSIONS` and `PREFERENCES` entries on its own.

So at this point I'd lean towards removing this file unless there any particular config wrt the default actions that we'd need to consider

Makes sense? cc @markocikos @izaera  maybe you're a bit familiar with permissions too

Reply:
Ya that makes sense to remove this file and keep the defaults. 👍